libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -829,3 +831,22 @@ void AC_WPNav::calc_scurve_jerk_and_jerk_time()
     }
     _scurve_jerk_time *= 2.0f;
 }
+
+// compute and set the horizontal error
+void AC_WPNav::compute_track_xy_error(const Vector3f &curr_pos) {
+    Vector3f track_pos_desired = _destination - _origin;
+    const Vector3f track_pos_curr = curr_pos - _origin;
+    Vector3f track_pos_error {0, 0, 0};
```

Comment:
No need to initialise the vector3f to 0 I think.

Reply:
Fixed. Ahh I didn't know that was the case for vector3f, but it is.